R/utils-deps.R

```diff
@@ -45,15 +45,19 @@ component_dependency_sass <- function(theme) {
   }
 }
 
-component_dependency_sass_ <- function(theme) {
+component_dependency_sass_layer <- function() {
```

Comment:
@cpsievert You probably don't want to call this `_sass_layer`. If so, can you propose a new name?